### PR TITLE
Rt600: Add information to connect an external debug probe

### PR DESCRIPTION
**Pull request by mmahadevan108**
Update documentation to include information on how to connect an external debug probe

### CODE REVIEW

File: boards/arm/mimxrt685_evk/doc/index.rst [hunk 1]
```diff
@@ -180,12 +180,21 @@ configured by default to use the LPC-Link2.
...
 Install the :ref:`jlink-debug-host-tools` and make sure they are in your search
-path.
+path. Install jumpers JP17, JP18 and JP19, to connect the SWD signals from onboard
+debug circuit.  These jumpers or installed by default.
 
 Follow the instructions in :ref:`lpclink2-jlink-onboard-debug-probe` to program
 the J-Link firmware. Please make sure you have the latest firmware for this
 board.
 
+:ref:`j-link-external-debug-probe`
```
Comment (hakehuang) [inline boards/arm/mimxrt685_evk/doc/index.rst:190]: this link exist?
```
/home/runner/work/zephyr/zephyr/doc/_build/src/boards/arm/mimxrt685_evk/doc/index.rst:190:undefined label: j-link-external-debug-probe
```

**Review by hakehuang — DISMISSED** [review]

File: boards/arm/mimxrt685_evk/doc/index.rst [hunk 2]
```diff
@@ -180,12 +180,21 @@ configured by default to use the LPC-Link2.
 -----------------------------------------
 
 Install the :ref:`jlink-debug-host-tools` and make sure they are in your search
-path.
+path. Install jumpers JP17, JP18 and JP19, to connect the SWD signals from onboard
+debug circuit.  These jumpers or installed by default.
```
Comment (MaureenHelm) [inline boards/arm/mimxrt685_evk/doc/index.rst:184]: s/or/are/

File: boards/arm/mimxrt685_evk/doc/index.rst [hunk 3]
```diff
@@ -180,12 +180,21 @@ configured by default to use the LPC-Link2.
...
 Follow the instructions in :ref:`lpclink2-jlink-onboard-debug-probe` to program
 the J-Link firmware. Please make sure you have the latest firmware for this
 board.
 
+:ref:`jlink-external-debug-probe`
+-----------------------------------------
+
+Install the :ref:`jlink-debug-host-tools` and make sure they are in your search
+path. Remove jumpers JP17, JP18 and JP19, to disconnect the SWD signals from onboard
+debug circuit.  These jumpers or installed by default. Connect the J-Link probe
```
Comment (MaureenHelm) [inline boards/arm/mimxrt685_evk/doc/index.rst:195]: s/or/are/

**Review by MaureenHelm — CHANGES_REQUESTED** [review]

File: boards/arm/mimxrt685_evk/doc/index.rst (hunk 3 above)
Comment (mmahadevan108) [reply boards/arm/mimxrt685_evk/doc/index.rst]: Thanks for catching this

File: boards/arm/mimxrt685_evk/doc/index.rst (hunk 1 above)
Comment (mmahadevan108) [reply boards/arm/mimxrt685_evk/doc/index.rst]: Thank you, I have fixed it.

**Review by MaureenHelm — APPROVED** [review]

**Review by galak — APPROVED** [review]

**Pull request merged**